src/libraries/Native/Windows/System.IO.Compression.Native/zlib-intel/inflate.c

```diff
@@ -1540,7 +1540,7 @@ z_streamp source;
     struct inflate_state FAR *state;
     struct inflate_state FAR *copy;
     unsigned char FAR *window;
-    unsigned wsize;
+    unsigned wsize = 0;
```

Comment:
Does this need to be upstreamed to https://github.com/jtkukunas/zlib ?

Reply:
Well it depends. Technically this isn't needed. The CA just isn't smart enough to understand it is properly set before use - a case of too many branches between set and use. I could have simply suppressed the warning inline but in this case it was cleaner to explicitly initialize it instead of putting in a pragma.

Reply:
We refresh these sources from upstream every once in a while. We try to push all changes we make in these sources upstream, so that we do not have to reapply the changes during every refresh.

Reply:
>We refresh these sources from upstream every once in a while.

Didn't know that. Will submit a PR.

Reply:
Done - https://github.com/jtkukunas/zlib/pull/30.

Reply:
@danmoseley Good call. This change has been merged upstream.